plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/TrinoNessieCatalog.java

```diff
@@ -253,6 +253,20 @@ public Transaction newCreateTableTransaction(
                 Optional.of(session.getUser()));
     }
 
+    @Override
+    public Transaction newCreateOrReplaceTableTransaction(ConnectorSession session, SchemaTableName schemaTableName, Schema schema, PartitionSpec partitionSpec, SortOrder sortOrder, String location, Map<String, String> properties)
```

Comment:
```suggestion
    public Transaction newCreateOrReplaceTableTransaction(
            ConnectorSession session, 
            SchemaTableName schemaTableName, 
            Schema schema, 
            PartitionSpec partitionSpec, 
            SortOrder sortOrder, 
            String location, 
            Map<String, String> properties)
```

Same for the other implementations